[2024-02-28 14:37:01,662][lightning_fabric.utilities.seed][INFO] - Seed set to 333
[2024-02-28 14:37:02,024][pytorch_lightning.utilities.rank_zero][INFO] - GPU available: True (cuda), used: True
[2024-02-28 14:37:02,071][pytorch_lightning.utilities.rank_zero][INFO] - TPU available: False, using: 0 TPU cores
[2024-02-28 14:37:02,071][pytorch_lightning.utilities.rank_zero][INFO] - IPU available: False, using: 0 IPUs
[2024-02-28 14:37:02,071][pytorch_lightning.utilities.rank_zero][INFO] - HPU available: False, using: 0 HPUs
[2024-02-28 14:37:50,079][pytorch_lightning.accelerators.cuda][INFO] - LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [2]
[2024-02-28 14:37:50,081][pytorch_lightning.callbacks.model_summary][INFO] - 
  | Name    | Type       | Params
---------------------------------------
0 | encoder | Sequential | 9.6 K 
1 | decoder | Sequential | 9.7 K 
---------------------------------------
19.3 K    Trainable params
0         Non-trainable params
19.3 K    Total params
0.077     Total estimated model params size (MB)
[2024-02-28 14:38:06,865][pytorch_lightning.utilities.rank_zero][INFO] - `Trainer.fit` stopped: `max_epochs=2` reached.
[2024-02-28 14:38:06,957][pytorch_lightning.utilities.rank_zero][INFO] - Restoring states from the checkpoint path at /homes/o23gauvr/Documents/thèse/code/FASCINATION/outputs/2024-02-28/14-36-58/AE_without_AP_training/no_pool_4/checkpoints/val_loss=0.12-epoch=01.ckpt
[2024-02-28 14:38:06,967][pytorch_lightning.accelerators.cuda][INFO] - LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [2]
[2024-02-28 14:38:06,972][pytorch_lightning.utilities.rank_zero][INFO] - Loaded model weights from the checkpoint at /homes/o23gauvr/Documents/thèse/code/FASCINATION/outputs/2024-02-28/14-36-58/AE_without_AP_training/no_pool_4/checkpoints/val_loss=0.12-epoch=01.ckpt
